tests/ert/unit_tests/analysis/snapshots/test_es_update/test_update_report/0-misfit_preprocess3/update_log

```diff
@@ -1,212 +1,212 @@
-------------  -------------------  -----  -----  -----  -----  ------  -----  ------
-FOPR          2010-01-10T00:00:00  0.002  0.100  5.657  0.566   0.076  0.105  Active
```

Comment:
To decrease your diff you could probably just fix the formatting here 😅 Not a big deal though, see that it is only the formatting that changed.